client/src/components/MapMarker/UserMarker.js

```diff
@@ -4,14 +4,15 @@ import { View, Text } from 'react-native';
 
 import styles from './styles';
 
-const UserMarker = ({ name, status, destination }) => (
+const UserMarker = ({ name, status, destination, locationTime }) => (
```

Comment:
test for this? if you have one already it doesn't show in this diff which means you're not covering `locationTime`.

Reply:
?

Reply:
the test was updated to include this.